test/core/test/happy-dom.test.ts

```diff
@@ -11,8 +12,19 @@ import { expect, it, vi } from 'vitest'
 declare global {
   // eslint-disable-next-line no-var
   var __property_dom: unknown
+  // eslint-disable-next-line no-var
+  var happyDOM: unknown
 }
 
+it('defaults URL to localhost:3000', () => {
+  expect(location.href).toBe('http://localhost:3000/')
+})
+
+it('accepts custom environment options', () => {
+  // default is false
+  expect((window.happyDOM as any).settings.disableCSSFileLoading).toBe(true)
+})
```

Comment:
Because we leverage on passing envrionment options via JSDoc code block, I didn't find a way where I could test passing the option vs not passing.